fxapom/pages/marionette/base.py

```diff
@@ -0,0 +1,16 @@
+
+from fxapom.fxapom import TIMEOUT
+from page import Page
```

Comment:
We should be consistent with our imports. Try: `from fxapom.pages.page import Page`